mainnet/singularity/cluster_config_template.json

```diff
@@ -90,10 +90,10 @@
         "DIFF_DIVIDER": 10000,
         "WINDOW_SIZE": 512,
         "TOTAL_STAKE_PER_BLOCK": 1000000000000000000000000,
-        "BOOST_TIMESTAMP": 1646064000,
+        "BOOST_TIMESTAMP": 1649044800,
```

Comment:
Since it took us one week to test devnet, should we extend the timestamp to another week?